main.js

```diff
@@ -185,23 +185,34 @@ function openCommentTextFn() {
 // }
 
 
-function createPost() {
+
+
+// Creating border Phone
+const borderPhone = document.createElement('div');
+borderPhone.className = "phone";
+document.body.append(borderPhone)
+
+//Creating function for the post
+function createPost(avatarSrc, song, backgroundImg, buttonId, openCommentText, comment, ) {
     let isFilled = false;
     let isOpen = true;
+
+    // creating function for the like button
     function likePostFn() {
-        const img = document.getElementById('like_btn');
+        const img = document.getElementById(buttonId);
         if(!isFilled) { 
-            img.src = './coeur.png';
+            img.src = './images/coeur.png';
             isFilled = true;
         } else { 
-            img.src = './heart.png';
+            img.src = './images/heart.png';
            isFilled = false;
         };
     
     };
+
+    //creating function for open comment text
     function openCommentTextFn() {
-        const openText = document.getElementById('texte');
-    
+        const openText = document.getElementById(openCommentText);
```

Comment:
Why this function is inside the `createPost` fn, the same question is for all the functions that you created inside the `createPost` fn